.gitignore

```diff
@@ -3,9 +3,6 @@ __pycache__/
 *.py[cod]
 *$py.class
 
-# C extensions
-*.so
-
```

Comment:
This might make a merge conflict with your other pull request:
https://github.com/cha2251/rit-player-piano/pull/11/files

Reply:
I branched that one off of this, so it should be good. But if not I'll fix the conflict in that one